database/buildSchema/05_trigger_queue.sql

```diff
@@ -11,7 +11,7 @@ CREATE TYPE public.trigger AS ENUM (
     'ON_REVIEW_ASSIGN',
     'ON_REVIEW_UNASSIGN',
     'ON_REVIEW_REASSIGN',
-    'ON_REVIEW_SELF_ASSIGN',
+    'ON_REVIEW_SELF_ASSIGN', -- Not used (not removed for consitency with previous snapshots)
```

Comment:
You can safely remove this, as long as you write migration code to deprecate the value. Old snapshots will load with the database script they were saved with, so will have this value included. Then they'll run migration to remove it. (Migration code will need to also *change* any existing to "ON_REVIEW_ASSIGN" otherwise it won't let you remove the value)